Makefile.third

```diff
@@ -38,10 +38,13 @@ $(TURBOJPEG_LIB) $(JPEG_LIB): $(THIRDPARTY_DIR)/libjpeg-turbo/CMakeLists.txt
 	cd $(JPEG_BUILD_DIR) && \
 		$(CMAKE) \
 		-DCMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
-		$(if $(findstring armv6, $(ARM_ARCH)),-DWITHOUT_SIMD:BOOL=ON,) \
+		$(if $(findstring arm, $(ARM_ARCH)),-DWITHOUT_SIMD:BOOL=ON,) \
```

Comment:
Nope, I want SIMD enabled on armv7 ;).

Reply:
If you had a build issue related to ARM detection, that's to be addressed in the crappy CMakeFile for libjpeg-turbo (there's a bunch of eInk & Android specific cases, it should be somewhat obvious, ping me if that looks too arcane ;)).

Reply:
Namly, you might need to add PRST the ginormous ENV(...) branch ;).